tools/publish-crates.sh

```diff
@@ -37,7 +37,7 @@ for crate in "${CRATES[@]}" ; do
 	if [ $DRY_RUN == 1 ] ; then
 		cargo publish --dry-run
 	else
-		cargo publish
+		cargo publish --allow-dirty
```

Comment:
Oops! I should not have left this line change in. Although we will probably technically need it for every single release unless someone removes this line permanently or we find another way to work around the error you get when attempting to push it to crates.io.

```rust
#![doc = include_str!("../../../../docs/incremental-joins.md")]
```
**Source:** https://github.com/clockworklabs/SpacetimeDB/blob/master/crates/core/src/subscription/subscription.rs#L24

**Error Message:**
```
error: couldn't read src/subscription/../../../../docs/incremental-joins.md: No such file or directory (os error 2)
  --> src/subscription/subscription.rs:24:10
   |
24 | #![doc = include_str!("../../../../docs/incremental-joins.md")]
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include_str` (in Nightly builds, run with -Z macro-backtrace for more info)
   ```

Thoughts?